GETSTARTED.md

```diff
@@ -2,10 +2,18 @@
 title: "Getting started using R"
 ---
 
-* [Install R](http://cran.r-project.org/)
-* [R Reference Card](http://cran.r-project.org/doc/contrib/Short-refcard.pdf)
-* [R tutorials CRAN page](http://cran.r-project.org/other-docs.html)
-* [RStudio](http://www.rstudio.com/)
-* [Comprehensive list of resources for getting started in R](http://scs.math.yorku.ca/index.php/R:_Getting_started_with_R) 
-* [swirl: learn R interactively](http://swirlstats.com/)
-* [Loading packages](LOAD_PACKAGES.html)
+If you are new to R, here are some resources that can get you started:
+
+
+# Software
+
+- [Install R](http://cran.r-project.org/)
+- [Install RStudio](http://www.rstudio.com/)
+- [swirl: learn R interactively](http://swirlstats.com/)
+
+# Reference
+
+- [A few words for R beginners](https://www.biostars.org/p/123728/?utm_content=buffereadb1&utm_medium=social&utm_source=twitter.com&utm_campaign=buffer)
```

Comment:
I think you want to remove the marketing/tracking parameters in the URL, i.e., everything after the `?` (and including the `?` itself).